### DISCUSSION TITLE
Can you add a shortcut to adjust the font size?

### DISCUSSION THREAD

**Discussion by mrzhu666** [Eugeny/tabby#5296] [General]
increase/decrease font size replaces zoom in/out may be better?

**Comment by Eugeny** [comment]
Is there a difference?

**Reply by mrzhu666** [reply]
when I fixed font size and zoom out,there are blank areas on the right and bottom
![image](https://user-images.githubusercontent.com/47910447/147399387-6e4ffa75-e8d5-4ad3-ae67-b701c279ec43.png)

![image](https://user-images.githubusercontent.com/47910447/147399384-b26ee4af-2fa2-4c86-9a73-9e928689a04b.png)

**Reply by mrzhu666** [reply]
Adjusting font size directly is not the case.Might it be better to adjust the font size instead?
![image](https://user-images.githubusercontent.com/47910447/147399455-e30f5c7d-f3b6-44a9-95b4-854b632840d4.png)
![image](https://user-images.githubusercontent.com/47910447/147399461-408b43f4-60f5-4422-be9f-837291f46753.png)
